cicecore/drivers/standalone/cice/CICE_RunMod.F90

```diff
@@ -386,6 +480,7 @@ subroutine ice_step
          call accum_hist (dt)               ! history file
          call ice_timer_stop(timer_hist)    ! history
 
+         call ice_barrier()
```

Comment:
and this one is not protected by `OPM_TIMERS`. is that what we want ?

Reply:
Same.